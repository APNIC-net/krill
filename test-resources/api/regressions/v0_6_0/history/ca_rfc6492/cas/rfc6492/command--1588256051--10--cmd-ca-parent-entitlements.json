{
  "actor": "krill",
  "time": "2020-04-30T14:14:11.665054Z",
  "handle": "rfc6492",
  "version": 13,
  "sequence": 10,
  "details": {
    "update_resource_classes": [
      "ta",
      {}
    ]
  },
  "effect": {
    "events": [
      13
    ]
  }
}