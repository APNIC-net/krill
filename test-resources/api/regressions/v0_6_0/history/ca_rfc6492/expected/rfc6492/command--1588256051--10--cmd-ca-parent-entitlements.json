{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:14:11.665054Z",
    "handle": "rfc6492",
    "version": 13,
    "sequence": 10,
    "details": {
      "update_resource_classes": [
        "ta",
        {}
      ]
    },
    "effect": {
      "events": [
        13
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "rfc6492",
        "version": 13,
        "details": {
          "resource_class_removed": [
            "0",
            {
              "ca_repo": "rsync://localhost/repo/rfc6492/0",
              "added": [],
              "updated": [],
              "withdrawn": [
                {
                  "name": "17143836B8F6D11DCB53CD49B5D40C61B8E9B813.mft",
                  "hash": "103f1492f73a85ac4f03f1fe7bbee129f8235cbe4ae9d013e6f5627e5938d9ec"
                },
                {
                  "name": "17143836B8F6D11DCB53CD49B5D40C61B8E9B813.crl",
                  "hash": "4932f3e3f2b1a53a6926c6e9dbea98eda6265536f3906a523b4c377873053f0d"
                }
              ]
            },
            "ta",
            [
              {
                "class_name": "0",
                "key": "17143836B8F6D11DCB53CD49B5D40C61B8E9B813"
              }
            ]
          ]
        }
      }
    ]
  }
}